charts/dvpe-deployment-gloo/templates/secret.yaml

```diff
@@ -33,6 +33,26 @@ spec:
 
 {{- end }}
 
+{{- if and .Values.externalSecrets.oidc.key .Values.gloo.authConfig.spec.configs.clientCredentialsPlugin.config.clientId }}
+---
+apiVersion: 'kubernetes-client.io/v1'
+kind: ExternalSecret
+metadata:
+  name: {{ $serviceName }}-oidc-m2m-secrets
+  namespace: {{ .Release.Namespace }}
+spec:
+  backendType: secretsManager
+  template:
+    metadata:
+      annotations:
+        resource_kind: '*v1.Secret'
+  data:
+    - key: {{ .Values.externalSecrets.oidc.key }}
```

Comment:
Same here, change `oidc` to `oauth2`

Reply:
hmm, this is defined in `values.yaml` and is not used only by the` clientCredentials`

Reply:
should I change it or keep it?

Reply:
I've just read the comment below :)

Reply:
done